app/models/collection.rb

```diff
@@ -6,20 +6,20 @@ class Collection < ActiveRecord::Base
 
   has_and_belongs_to_many :projects
   has_many :collections_subjects,
-    -> { order(id: :desc) },
-    dependent: :destroy
+           -> { order(id: :desc) },
```

Comment:
Just a note: As it's a long running code base with lots of lint errors in it, I avoid fixing these kinds of lint errors in my IDE as they make the PR diff super noisy. Most of this class PR diff is lint fixes, not new behaviour.

Reply:
Keeping this in mind moving forward 👍